packages/react/src/hooks/useTheme.ts

```diff
@@ -27,3 +28,11 @@ export const useTheme = (): WebTheme => {
   const context = React.useContext(ThemeContext);
   return getThemeFromContext(context);
 };
+
+/**
+ * Internal use only
+ */
+export const useColorMode = (): ColorMode | undefined => {
```

Comment:
Added this hook to see when the color mode has changed, made it internal only for now since it is just used in liveness to see when the color mode has changed